### ISSUE TITLE
Delete branch button overflow the box

### ISSUE THREAD

**Issue by ghost** [go-gitea/gitea#21310]
### Screenshots

![Delete Branch Button](https://user-images.githubusercontent.com/76462613/193409014-d740e588-f492-49f3-9815-91e915bd69d6.jpeg)
![](https://user-images.githubusercontent.com/76462613/193409338-46c289d3-4b22-4737-83e6-5584a83ed315.jpeg)

### Gitea Version

 1.18.0+dev-421-geaa561145

### Can you reproduce the bug on the Gitea demo site?

Yes

### Browser Version

Edge

**Comment by neonn** [comment]
Hi, I thought this issue was interesting, so I thought I'd dive a little deeper:

When viewing the Branches overview page, and viewing at a smaller resolution than 1920 by 1080 or a DPI larger than 1 (i.e. the breakpoint trigger at `@media only screen and (min-width: 992px) and (max-width: 1199.98px)` or smaller), the action buttons to the right of each branch overflow their container. In addition, the "Create Pull Request" button (because it is actually a link) shows a single ellipsis. 

I've also tested this on 1.17.0, using Arch Linux as my OS.

I've also tested these browsers:
- Chromium ("Version 105.0.5195.125 (Official Build) Arch Linux (64-bit)")
- Firefox ("106.0b3 (64-bit)")

Both browsers show the same issue.

@rock2dust can I suggest a title change, to something like "Repository Branches Overview - Action Button Overflow"? That way, it's a bit clearer what this issue intends to fix. Good find though! 😃
